workflows/prognostic_c48_run/runtime/factories.py

```diff
@@ -51,10 +59,48 @@ def get_tendency_prescriber(
     if config.tendency_prescriber is None:
         return None
     else:
+        prescriber_config = config.tendency_prescriber
+        mapper_config = prescriber_config.mapper_config
+        tendency_variables = list(prescriber_config.variables.values())
+        if communicator.rank == 0:
+            logger.debug(f"Opening tendency override from: {mapper_config}")
+        mapper_function = _get_mapper_function(
+            mapper_config,
+            tendency_variables,
+            prescriber_config.reference_initial_time,
+            prescriber_config.reference_frequency_seconds,
+        )
         return TendencyPrescriber(
-            config.tendency_prescriber,
             state,
             communicator,
             timestep,
+            prescriber_config.variables,
+            mapper_function,
+            limit_quantiles=prescriber_config.limit_quantiles,
             diagnostic_variables=set(config.diagnostic_variables),
         )
+
+
+def _get_mapper_function(
+    mapper_config: loaders.MapperConfig,
+    tendency_variables: Sequence[str],
+    initial_time: Optional[str] = None,
+    frequency_seconds: float = 900.0,
+) -> Callable[[cftime.DatetimeJulian], State]:
+
+    mapper = mapper_config.load_mapper()
+
+    def mapper_function(time: cftime.DatetimeJulian) -> State:
+        timestamp = vcm.encode_time(time)
+        ds = mapper[timestamp]
+        return {var: ds[var] for var in tendency_variables}
+
+    if initial_time is not None:
+        initial_time = label_to_time(initial_time)
+        return_func = time_interpolate_func(
+            mapper_function, timedelta(seconds=frequency_seconds), initial_time
+        )
+    else:
+        return_func = mapper_function
```

Comment:
Nit...but I personally prefer to eagerly return things in these kinds of statements rather than introducing a new symbol that could be modified later on.
```suggestion
        return time_interpolate_func(
            mapper_function, timedelta(seconds=frequency_seconds), initial_time
        )
    else:
        return mapper_function
```